### ISSUE TITLE
Default None type changes to unicode string "None"

### ISSUE THREAD

**Issue by kiccsepi** [robotframework/PythonLibCore#7]
https://github.com/robotframework/PythonLibCore/blob/025ce88420a354ead7225774330d3624e7bbb508/src/robotlibcore.py#L107

Greetings!

I have experienced the following issue with DynamicCore:
Env: Robot Framework 3.0.4 (Python 2.7.15 on win32)

```
@keyword
def example(self, p1, p2=None, p3=False):
    import logging
    logging.debug(p2)
    logging.debug(type(p2))
    logging.debug(p3)
    logging.debug(type(p3))
```
OK:
`Example | foobar |`
20180920 11:40:19.188 : DEBUG : None
20180920 11:40:19.189 : DEBUG : <type 'NoneType'>
20180920 11:40:19.189 : DEBUG : False
20180920 11:40:19.189 : DEBUG : <type 'bool'>

OK:
`Example | foobar | p2=${None}`
20180920 11:42:47.513 : DEBUG : None
20180920 11:42:47.514 : DEBUG : <type 'NoneType'>
20180920 11:42:47.514 : DEBUG : False
20180920 11:42:47.515 : DEBUG : <type 'bool'>

NOK:
`Example | foobar | p3=${True}`
**20180920 11:43:23.717 : DEBUG : None
20180920 11:43:23.717 : DEBUG : <type 'unicode'>**
20180920 11:43:23.718 : DEBUG : True
20180920 11:43:23.718 : DEBUG : <type 'bool'>

As you can see, the default value is not correct for the p2 in the last case.

Without the dynamic core, e.g: Built-In libraries are not affected:
`Library | OperatingSystem`
`List Directory | ${CURDIR} | absolute=${True}`
20180920 11:54:03.198 : DEBUG : None
20180920 11:54:03.198 : DEBUG : <type 'NoneType'>
20180920 11:54:03.198 : DEBUG : True
20180920 11:54:03.198 : DEBUG : <type 'bool'>

Cheers from Budapest,
Rábel, Sándor

**Comment by pekkaklarck** [comment]
This problem is actually in Robot's side but it is fixed in RF 3.1. See robotframework/robotframework#2930 for details.

**Comment by kiccsepi** [comment]
Great! Thanks for the fix!
